batch/launch_job.py

```diff
@@ -27,30 +27,39 @@
               help="The name of the AWS Batch Job Definition to use for the job")
 @click.option("-q", "--job-queue", "batch_job_queue", type=str, default="Batch-CovidPipeline", show_default=True,
               help="The name of the AWS Batch Job Queue to use for the job")
-def launch_batch(config_file, num_jobs, sims_per_job, dvc_target, s3_input_bucket, s3_output_bucket, batch_job_definition, batch_job_queue):
+@click.option("-p", "--parallelize-scenarios", "parallelize_scenarios", type=bool, default=False, show_default=True,
```

Comment:
would this be better as a scenarios/job option (default is MAX)? if i'm running usa, i'll set it to 1 because it's a hefty job, but if i'm running say louisana and there are 100 scenarios, i may make it 10 because each scenario finishes in 2 mins and i want sufficient utilization per machine. maybe this is overkill...